docs/supported-data-types-in-carbondata.md

```diff
@@ -51,4 +51,5 @@
 
   * Other Types
     * BOOLEAN
+    * [BINARY](./supported-binary-in-carbondata.md)
```

Comment:
Just remove the `supported-binary-in-carbondata.md`

Reply:
done